src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Utils/MockFormTestUtils.cs

```diff
@@ -34,5 +36,27 @@ public static Mock<IFormFile> CreateFormFileMock(string filename, string content
 
             return formFile;
         }
+
+        public static IFormFile CreateFormFileFromResource(string fileName, string? newFileName = null)
+        {
+            var filePath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!,
+                "Resources" + Path.DirectorySeparatorChar + fileName);
```

Comment:
Was half expecting this to break the tests, but obviously not. Seems that "GetExecutingAssembly().Location" is refers not to Common.Tests, but the project from which this method is executed.